pkg/asset/internal/templates.go

```diff
@@ -1192,14 +1193,17 @@ spec:
               value: "Always"
             - name: FELIX_IPINIPENABLED
               value: "true"
+            - name: FELIX_TYPHAK8SSERVICENAME
+              valueFrom:
+                configMapKeyRef:
+                  name: calico-config
+                  key: typha_service_name
             - name: NODENAME
               valueFrom:
                 fieldRef:
                   fieldPath: spec.nodeName
             - name: IP
-              valueFrom:
-                fieldRef:
-                  fieldPath: status.podIP
```

Comment:
I think there was a reason this was here. Probably something to do with vagrant examples and such. Why was this changed? We should probably test the local VM examples.

Reply:
This is part of the v3 changes: https://github.com/projectcalico/calico/pull/1588/files

Reply:
Okay thanks 👍